csvqb/csvqb/writers/qbwriter.py

```diff
@@ -425,11 +441,15 @@ def _define_csvw_column_for_qb_column(
         if column.output_uri_template is not None:
             # User-specified value overrides our default guess.
             csvw_col["valueUrl"] = column.output_uri_template
+        elif isinstance(column.component, QbAttributeLiteral):
+            pass
```

Comment:
This makes me think we should add a test to say that if a user declares that the column has a `QbAttributeLiteral` component then they get a validation error when they set an `output_uri_template` on it. I think that can be part of a separate ticket though.

Reply:
https://github.com/GSS-Cogs/csvwlib/issues/130